client/src/components/SystemStatus/SystemStatus.jsx

```diff
@@ -51,17 +51,12 @@ const Layout = ({ hostname }) => {
   return (
     <>
       <Section
-        className={styles['section']}
-        contentLayoutName="oneColumn"
+        className={styles['root']}
+        contentLayoutName="twoColumn"
+        contentClassName={styles['layout']}
         content={
-          <>
-            <SectionTableWrapper
-              className={styles['sysmon-content']}
-              contentShouldScroll
-            >
-              <Sysmon system={selectedSystem?.hostname}></Sysmon>
-            </SectionTableWrapper>
-
+          <div className={styles['panel-1']}>
+            <Sysmon system={selectedSystem?.hostname} />
```

Comment:
The `<SectionTableWrapper>` is used to support scrolling a table. This table need not scroll.